lib/matplotlib/quiver.py

```diff
@@ -87,21 +87,28 @@
 angles : {'uv', 'xy'} or array-like, default: 'uv'
     Method for determining the angle of the arrows.
 
-    - 'uv': Arrow direction in screen coordinates. Use this if the arrows
-      symbolize a quantity that is not based on *X*, *Y* data coordinates.
+    - 'uv': Arrow directions are based on screen coordinates (the coordinate
+       system of the plot, not the data). This is useful when the arrows represent 
+       a quantity that is not directly tied to the x and y data coordinates, such as
+       abstract physical quantities (e.g., velocities, forces).
 
       If *U* == *V* the orientation of the arrow on the plot is 45 degrees
       counter-clockwise from the horizontal axis (positive to the right).
 
     - 'xy': Arrow direction in data coordinates, i.e. the arrows point from
-      (x, y) to (x+u, y+v). Use this e.g. for plotting a gradient field.
+      (x, y) to (x+u, y+v). This is ideal for vector fields or gradient plots 
+      where the arrows should directly represent movements or gradients in the 
+      x and y directions.
 
-    - Arbitrary angles may be specified explicitly as an array of values
+    - 'Array-like':Arbitrary angles may be specified explicitly as an array of values
```

Comment:
The quotation marks above are for indicating strings, and we're not addressing the "Array-like" here. If anything this should be `- array-like: Arbitrary angles ...`, but I think it's also clear without.